docs/model/simulation_outputs.rst

```diff
@@ -50,14 +50,16 @@ KPI that describe the costs of the overall energy system do not have suffixes. T
 Economic KPI
 ############
 
+All the KPI related to costs described below are provided in net present value.
+
 .. include:: outputs/costs_total.inc
 
-The Net present cost (NPC) is the present value of all the costs associated with installation, operation,
+The Net present costs (NPC) is the present value of all the costs associated with installation, operation,
 maintenance and replacement of energy technologies comprising the sector-coupled system over the project lifetime,
 minus the present value of all the revenues that it earns over the project lifetime.
 The capital recovery factor (CRF) is used to calculate the present value of the cash flows.
 
-** The content of this section was copied from the conference paper handed in to CIRED 2020**
+**The content of this section was copied from the** `conference paper <http://doi.org/10.5281/zenodo.4449918>`_ **handed in to CIRED 2020**
```

Comment:
I think I would rather reformulate this paragraph. Probably not use cites for now.